src/routes/console/project-[project]/auth/templates/smsLoginTemplate.svelte

```diff
@@ -0,0 +1,46 @@
+<script lang="ts">
+    import SmsTemplate from './smsTemplate.svelte';
+    import LocaleOptions from './localeOptions.svelte';
+    import type { Models } from '@appwrite.io/console';
+    import { page } from '$app/stores';
+    import { loadSmsTemplate } from './+page.svelte';
+    import { baseSmsTemplate, smsTemplate } from './strote';
+    import { addNotification } from '$lib/stores/notifications';
+    import { Id } from '$lib/components';
+
+    export let localeCodes: Models.LocaleCode[];
+
+    const projectId = $page.params.project;
+    let locale = 'en';
```

Comment:
we should use constants for these strings

Reply:
the locale changes so it can't be a constant :smile: